copy-kibana-dashboard.rb

```diff
@@ -72,6 +76,7 @@ def parse_args
     ['--to-host', GetoptLong::OPTIONAL_ARGUMENT],
     ['--to-port', GetoptLong::OPTIONAL_ARGUMENT],
     ['--to-index', GetoptLong::OPTIONAL_ARGUMENT],
+    ['--to-saved-search-index', GetoptLong::REQUIRED_ARGUMENT],
```

Comment:
REQUIRED_ARGUMENT seems to mean that the option requires and argument. So i guess some of the other options should be REQUIRED_ARGUMENT also, or?